cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -95,6 +95,9 @@ function start-kube-apiserver {
   if [[ -n "${TLS_CIPHER_SUITES:-}" ]]; then
     params+=" --tls-cipher-suites=${TLS_CIPHER_SUITES}"
   fi
+  if [[ -e "${KUBE_HOME}/bin/gke-internal-configure-helper.sh" ]]; then
+    params+=" $(gke-kube-apiserver-internal-sni-param)"
```

Comment:
I'm not that familiar with the ins and outs of these configuration scripts other than what I've had to do to debug some gnarly issues.  Does the existence of the script necessarily indicate that it has been `source`d by the time we're here?  This feels like it's begging to throw a `command not found` at us.

My initial reaction was to suggest we `source` the `*configure-helper.sh` here, since it should be safe to `source` in this `if` block even if it has already been `source`d before.  But looking around elsewhere, there are a other commands (i.e., `append-param-if-not-present`, `get-metadata-value`) that are just assumed to be on the `PATH`, so maybe that worry is moot.

Reply:
It's sourced in https://github.com/Kubernetes/kubernetes/blob/master/cluster/gce/gci/configure.sh#L386